Tensile/LibraryLogic.py

```diff
@@ -1444,7 +1444,7 @@ def main(  config ):
     logicTuple = analyzeProblemType( problemType, problemTypes[problemType], \
         analysisParameters)
 
-    LibraryIO.configWriter(globalParameters["YAML"]).writeLibraryLogicForSchedule(globalParameters["WorkingPath"], \
+    LibraryIO.configWriter(globalParameters["LibraryFormat"]).writeLibraryLogicForSchedule(globalParameters["WorkingPath"], \
```

Comment:
I don't think we want this to follow LibraryFormat.  This should be YAML no matter what, correct?

Reply:
Yeah, you're right. The next commit makes both of the changes you've requested